src/utils/HashMultiMap.fs

```diff
@@ -98,7 +98,7 @@ type internal HashMultiMap<'Key,'Value>(n: int, hasheq: IEqualityComparer<'Key>)
                     firstEntries.[y] <- h
                     rest.[y] <- t
                 | _ -> 
-                    // note: broken invariant
+                    // NOTE: broken invariant
```

Comment:
Just remove the comment. The code can't be reached but we're not about to add an exception to the codebase just for the fun of it, so let's deal with it by removing the comment.